wordpress-mobile/WordPress-iOS
# 5273 - Remove checks for iPad when displaying alerts and popovers.

Fixes #5273

To test:

- Check the post list and page list section of the app
- Check on the iPhone, iPhone+ and iPad on the several multitasking methods.
- On page list selection the page options and see if the popover shows correctly
- On post list select the status filter and see if the popup shows correctly
- Add a self hosted site to the app,  change the password on the web and go to post list or page list and pull to refresh. Check if the alert error message shows correctly.

Needs review: @diegoreymendez
